htdocs/main.php

```diff
@@ -288,6 +288,15 @@ function tplFromRequest($param)
 
 $tpl_data['css'] = $config->getSetting('css');
 
+// Prints SQL Queries if a file with output exists
+if (isset($_SESSION['file_sql_queries'])) {
```

Comment:
I think `$_SESSION` should only be used when you need to keep state across multiple requests, and AFAIK that isn't needed to implement this feature.

As much as I hate to suggest using another global variable, I think it would be cleaner to just define a property on `$_GLOBALS`, like [this](http://stackoverflow.com/a/13530536/207041). It would probably be too much work right now to not rely on any globals.